community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/FunctionsTest.scala

```diff
@@ -147,6 +147,22 @@ class FunctionsTest extends DocumentingTestBase {
     )
   }
 
+  @Test def keys() {
+    testThis(
+      title = "KEYS",
+      syntax = "KEYS( node )",
```

Comment:
Seems this bit was missed when _keys()_ was expanded to work for relationships as well?